alf/experience_replayers/replay_buffer.py

```diff
@@ -30,8 +30,13 @@
 from .segment_tree import SumSegmentTree, MaxSegmentTree
 
 BatchInfo = namedtuple(
-    "BatchInfo",
-    ["env_ids", "positions", "importance_weights", "replay_buffer"],
+    "BatchInfo", [
+        "env_ids",
+        "positions",
+        "importance_weights",
+        "replay_buffer",
+        "discounted_return",
```

Comment:
I cant find any code that set a value for this.
Also, I feel BatchInfo is not the right place for this value since most of our code does not have easy access to it.
rollout_info may be a better place.

Reply:
Good catch, populated in get_batch, and then in preprocess_experience to rollout_info, and then train_step to policy_step.info, so td_loss can have access.